### DISCUSSION TITLE
Dell PowerEdge hangs

### DISCUSSION THREAD

**Discussion by Tampa** [ipxe/ipxe#1314] [General]
In reference to: https://github.com/netbootxyz/netboot.xyz/issues/1246

I have two machines that get stuck just like that. Something specific to this hardware or firmware that seems to cause it to just stop dead. Have already tried changing a bunch of things, but so far no luck. Happy to provide debug information, just not sure how.

**Comment by NiKiZe** [comment]
There isn't enough information to debug this in iPXE, or even see that this is an iPXE issue, so moving to discussions.

There is open issues on downloads for broadcom NICs on 10gbit links, but your downloads seems to be ok? Anyway, please use `ifstat` to see which iPXE driver is used.

Which iPXE binary are you using, I'm looking to know if you are booting `pcbios` or `efi`, this can be seen if you are using the original iPXE filename, such as `undionly.kpxe`, `ipxe.pxe`, `snponly.efi`, `ipxe.efi`.

In the context of your output, could you post your script that is used for the relevant screenshot?
Also before `boot` insert `imgstat` to show what is loaded, but also to determine exactly which line in the script it "hangs" on.

**Reply by Tampa** [reply]
Is there any way to turn on some verbose log to see what it does or tries to do? Or do I have to talk to netboot.xyz for that?

**Comment by Tampa** [comment]
The issue comes from netboot.xyz project, so I have no clue how it works internally. I am using bios mode and it should be loading the default ipxe that netboot.xyz is based on. I'm new to this whole thing so pardon my lack of knowledge. I'll see if I can run the ipxe shell and the commands.

Downloads are fine yes. I think the problem is likely to be at the point where it tries to inject? the binaries and load them. Been talking to netboot already to work out if there is some more verbose information I can turn on, they sent me here basically. Over at FOG I did see someone suggesting to use the undionly as well, so maybe that's the key, but we'll see. I'll update once I gathered the requested information.

**Reply by Tampa** [reply]
Using the ipxe shell of netboot. Running imgstat just gives me version.ipxe as return.

![image](https://github.com/user-attachments/assets/89464898-b393-4b64-876b-ebc1f7da2948)

Switching to the undionly version it fails to setup the net link.

![image](https://github.com/user-attachments/assets/f1a9c762-0bfe-492d-82ec-e6dfd9e4d454)

**Reply by Tampa** [reply]
I tried one of the netboot utilities, an ancient ubuntu live cd for repairing boot issues and that loaded fine. Tried memtest 7 before that, got stuck as well. Naturally I am now thinking did the newer installer deprecate some driver it needs to load from a pxe.

**Reply by Tampa** [reply]
memtest5.1 loads fine, but memtest7 image gets stuck. A clue, but I got no idea what's different between those versions.
